internal/cmd/commands/config/encryptdecrypt.go

```diff
@@ -142,7 +145,14 @@ func (c *EncryptDecryptCommand) Run(args []string) (ret int) {
 		kmsDefFile = strings.TrimSpace(c.flagConfigKms)
 	}
 
-	wrapper, err := wrapper.GetWrapperFromPath(kmsDefFile, "config")
+	wrapper, cleanupFunc, err := wrapper.GetWrapperFromPath(
```

Comment:
Reviewers: here's a file to look at. Note that cleanupFunc should never be nil; the reason I do a nil check is simply to avoid deferring another func that does error reporting if I don't need to. It could safely be taken out though. But also, nil checking is never bad. :-)